cucumber-messages/javascript/src/BinaryToMessageStream.ts

```diff
@@ -15,13 +15,14 @@ export default class BinaryToMessageStream<T> extends Transform {
 
   public _transform(chunk: any, encoding: string, callback: TransformCallback) {
     this.buffer = Buffer.concat([this.buffer, chunk])
-
-    while (true) {
+    let finished = false
```

Comment:
The only actual code change - seemed worthwhile to avoid ignoring the `no-constant-condition` rule which would otherwise be useful. This is covered by an existing test.